android/app/build.gradle

```diff
@@ -65,6 +65,14 @@ android {
         versionName "1.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        multiDexEnabled true // development purposes
+
+        // Define HTTP URL variable in BuildConfig
+        buildConfigField "String", "HTTP_URL", "\"http://192.168.1.106:8080\""
```

Comment:
How do I set custom buildConfig value on different computers?

Reply:
You can just change the value in build.gradle. Or do you mean something else?

Reply:
Say on 1 computer I want  to connect to localhost, but on another dev's computer they want to connect to robin.com.au.  We can't all be commiting to one build.gradle.

Reply:
So say in Python we just use environment variable to pass custom values.
eg.
```
API_HOST=localhost python script.py
API_HOST=robini.com.au python script.py # on another machine
```